Pipfile

```diff
@@ -0,0 +1,17 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+bless = {editable = true, file = "file:///home/gonzalo.peci/Workspace/src/bless"}
```

Comment:
This shouldn't use paths that are specific to your environment.

Reply:
Yep, I have some cleanup to do, but I wanted to know first if this would be something you were interested in merging.

Reply:
Yeah, I'm all for merging it.  While I've not looked at it closely in a long time, It would be good to also try and consolidate the HOST cert logic from:
https://github.com/lyft/bless/blob/lyft-host-certificate-lambda/bless/aws_lambda/bless_lambda.py

I'm in favor of having 2 different lambdas and ca keys, so you can maintain two clear IAM policies & trust conifgs.  For users who don't use kmsauth tokens, I wouldn't want servers being able to mint user ssh certs.